src/components/Home.js

```diff
@@ -0,0 +1,25 @@
+import "./Home.css";
+import bottleOne from "../images/bottle-one.png";
+import bottleTwo from "../images/bottle-two.png";
+
+export default function Home() {
+  return (
+    <div className="home">
```

Comment:
In React is normally a convention to name the class of the root element of a component after the name of the component (in this case `Home`) also using the capitalized H.

This is just a convention, and as long as you are consistent in your naming, is fine if you don't follow it, but it might make it easier for you to be consistent.

https://stackoverflow.com/questions/57221878/react-classname-naming-convention